<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CNC Solver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>440</width>
        <height>260</height>
       </size>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <widget class="QLineEdit" name="lineEditFilename">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>401</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonChooseFilename">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>20</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose directory</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Data output file:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxOverallCount">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Count</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxTaskCount">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>90</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Task count</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Task size</string>
       </property>
       <widget class="QSpinBox" name="spinBoxTaskSizeUpper">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxTaskSizeLower">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Uniform distribution</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>From</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Processing time</string>
       </property>
       <widget class="QSpinBox" name="spinBoxProcessingTimeUpper">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxProcessingTimeLower">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Uniform distribution</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>From</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </widget>
      <widget class="QSpinBox" name="spinBoxProcessorCount">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>90</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>90</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Processor count</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonCheckSchedule">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>170</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Check schedule</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="minimumSize">
       <size>
        <width>440</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>CNC Solver</string>
      </property>
      <widget class="QPushButton" name="pushButtonSolve">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Solve</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>311</width>
         <height>20</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
